README.md

```diff
@@ -23,13 +23,15 @@ size is acceptable for NCBI)
 
 After the files have been generated, the following files will be uploaded to external FTP Servers:
 
-* To the **NCBI FTP Server**, the "gene_reactomeXX-Y.xml" and "protein_reactomeXX.ft" files will be uploaded (using the 
+* To the **[NCBI FTP Server](ftp://ftp-private.ncbi.nih.gov)**, the "gene_reactomeXX-Y.xml" and "protein_reactomeXX.ft" files will be uploaded (using the 
 values provided in the configuration file to connect to the FTP Server - see the [Configuration](#configuration) 
-section below)
+section below).  NOTE: Access to the Reactome files requires user
```

Comment:
Could the 'NOTE' portion of this be bolded?

Reply:
Done: https://github.com/reactome/release-data-exporter/pull/4/commits/e93a869d6885b43d80de9553e008e9373aebfbc8